ui/admin/app/components/model/UpdateModelActive.tsx

```diff
@@ -0,0 +1,28 @@
+import { memo, useState } from "react";
+
+import { Model } from "~/lib/model/models";
+import { ModelApiService } from "~/lib/service/api/modelApiService";
+
+import { Switch } from "~/components/ui/switch";
+
+export const UpdateModelActive = memo(function UpdateModelActive({
```

Comment:
Purely curious, but why do we need memo here?

Reply:
Feel free to ignore that haha That is me trying to figure out what's causing the rerendering of the table unnecessarily, causing a flicker effect.